bin/cli

```diff
@@ -63,6 +64,9 @@ drupal_install() {
     install_configure_form.update_status_module='array(FALSE,FALSE)' \
     --yes
   set +e
+
+  echo "\$settings['config_sync_directory'] = '../config/sync';" >> sites/default/settings.php
+  drush config-set wxt_library.settings wxt.theme theme-gcweb --yes
```

Comment:
Can we only have the config_sync_dir line here, setting to gcweb might not be what others want.